common/src/main/java/com/skytouch/common/model/Product.java

```diff
@@ -0,0 +1,90 @@
+package com.skytouch.common.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+
+public class Product implements Serializable {
+
+
+    private Long id;
+
+    private String name;
+
+    private String description;
+
+    private Integer unitPrice;
+
+    private Integer quantityPerUnit;
+
+    public Product(String name, String description, Integer unitPrice, Integer quantityPerUnit) {
+        this.name = name;
+        this.description = description;
+        this.unitPrice = unitPrice;
+        this.quantityPerUnit = quantityPerUnit;
+    }
+
+    public Product() {
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public Integer getUnitPrice() {
+        return unitPrice;
+    }
+
+    public Integer getQuantityPerUnit() {
+        return quantityPerUnit;
+    }
+
+    public Integer setQuantityPerUnit() {
+        return quantityPerUnit;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setUnitPrice(Integer unitPrice) {
+        this.unitPrice = unitPrice;
+    }
+
+    public void setQuantityPerUnit(Integer quantityPerUnit) {
+        this.quantityPerUnit = quantityPerUnit;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof Product)) return false;
+        Product product = (Product) o;
+        return Objects.equals(name, product.name) &&
+                Objects.equals(description, product.description) &&
+                Objects.equals(unitPrice, product.unitPrice) &&
+                Objects.equals(quantityPerUnit, product.quantityPerUnit);
+    }
+
+    @Override
+    public int hashCode() {
+
+        return Objects.hash(name, description, unitPrice, quantityPerUnit);
+    }
+}
```

Comment:
Don't you consider it would be useful to override `toString()` method too?

Reply:
It's not neccessary, i'm going to remove it.

Reply:
If you have chance, check Effective Java Item 9

Reply:
I decided to re-add the overwrite of the equal and hashcode to compare the expected objects and those obtained in the unit tests.
I read the article and I consider it a good practice overwrite the hashcode whenever the equals is overwritten.